### DISCUSSION TITLE
Help setting up SAML

### DISCUSSION THREAD

**Discussion by brandonhon** [requarks/wiki#4834] [Help / Questions]
Is there anyone with experience setting up SAML SSO? I am trying to setup SAML with JumpCloud and there doesn't seem to be any documentation available for SAML setup on WikiJS.

A good start would be how or where can I generate provider metadata. I am using the container version of wikijs.

**Comment by Phil-hostwiki** [comment]
Use this script ([mellon_create_metadata.sh](https://raw.githubusercontent.com/UNINETT/mod_auth_mellon/master/mellon_create_metadata.sh)) to create a metadata file
`./mellon_create_metadata.sh https://wiki.example.com https://wiki.example.com/login/xxxxxx-xxxxx-xxxx-xxxx-xxxxxxx/callback`

PS: Replace the placeholder URLs with the one provided on your Wiki.js SAML setup page.

**Reply by cristobal-io** [reply]
Could you provide more instructions on how to set it up?

I am also using the docker approach.
